README.md

```diff
@@ -4,8 +4,7 @@
 
 ## Table of contents
 
-- [Table of contents](#table-of-contents)
-- [Setup](#Setup)
+- [Set up](#set-up)
```

Comment:
Wait a minute, this should actually be "Setup": "Setup" is a noun, while "Set up" is a verb. We're using nouns for the other titles, so this should be a noun too.

Reply:
https://github.com/eHealthAfrica/aether/pull/657 :see_no_evil: